File: Curso Logica de Programação Neri Neitzke/Algoritmos em Pascal/pascal/media_ler.pas
// Prof Neri Aldoir Neitzke
// www.INFORMATICON.com.br - videoaulasneri@gmail.com

program media_ler;
var
   nota1, nota2, calc_media: real;  //definição variáveis
begin
   write('Insira a primeira nota.: ');
   readln(nota1); //nota1 := 5;      //entradas
   write('Insira a segunda nota.: ');
   readln(nota2); //nota2 := 7;
   calc_media := (nota1 + nota2) /2; //processamento
   writeln('A media = ',calc_media);  //saída
   write('...................Final do Sistema..................... ');
   readln();
end.

File: Curso Logica de Programação Neri Neitzke/Algoritmos em Java/media_ler.java
// Prof Neri Aldoir Neitzke
// www.INFORMATICON.com.br - videoaulasneri@gmail.com
import javax.swing.JOptionPane;
class media_ler
{
   public static void main(String args[])
   {
      float nota1,nota2,calc_media; //criei as variáveis
                                    //DataInputStream = manipula dados de entrada try catch
      nota1 = Float.parseFloat(JOptionPane.showInputDialog("Digite a primeira nota"));
      nota2 = Float.parseFloat(JOptionPane.showInputDialog("Digite a segunda nota"));
      calc_media = (nota1 + nota2) / 2;
      //System.out.println("A media = "+calc_media);
      JOptionPane.showMessageDialog(null,"A media = "+calc_media);
   }
}